source/_posts/2020-01-15-release-104.markdown

```diff
@@ -74,28 +74,165 @@ Experiencing issues introduced by this release? Please report them in our [issue
+  ```yaml
+  axis:
+  ```
+
+- __WLED__ - Improves WLED pure white support for RGBW. When using WLED with RGBW strips, setting the color in Home Assistant to white, will turn off the RGB LEDs of the strip and use the W channel only. - ([@frenck] - [#30477]) ([wled docs]) 
+- __Timer__ - Fire restarted event on restarting active timer. This can break automations that should trigger on every timer start event. - ([@mnigbur] - [#30475]) ([timer docs]) 
+- __Vizio__ - Removed `suppress_warning` from config options for vizio component. The pyvizio component now suppresses insecure HTTPS request warnings in a way that doesn't affect global requests from other components, HA, etc. There is no need to globally suppress insecure warnings now. - ([@raman325] - [#30536]) ([vizio docs]) 
+
+  Example configuration:
+  ```yaml
+  media_player:
+    - platform: vizio
+      host: '<IP>:<PORT>'
+      access_token: '<AUTH_TOKEN>'
+      device_class: tv
+  ```
+
+- __Default groups__ - Entity integrations will no longer automatically maintain the hidden groups with all of their entities. The following groups are no longer automatically created and maintained:
+
+   - group.all_automations
+   - group.calendar
+   - group.all_covers
+   - group.all_devices
+   - group.all_fans
+   - group.all_lights
+   - group.all_locks
+   - group.all_plants
+   - group.remember_the_milk_accounts (???)
+   - group.all_remotes
+   - group.all_scripts
+   - group.all_switches
+   - group.all_vacuum_cleaners
+
+  The automation and script config panels have been updated to show and manage all automations/scripts. If you want to show cards in your UI with all entities for a single domain, use the auto-entities card. If you want to target all entities in a service call, use 'all' as value for entity_id.  - (  [@balloob] - [#23789]) ([automation docs]) ([calendar docs]) ([cover docs]) (  [device_tracker docs]) ([fan docs]) ([light docs]) ([lock docs]) ([plant   docs]) ([remember_the_milk docs]) ([remote docs]) ([script docs]) ([switch   docs]) ([vacuum docs])
+
+
+- __RFXtrx__ - Bump pyRFXtrx to 0.25. This update switches the commands for 'open' and 'close' for rfxtrx covers of the 'Rollertrol' type, as they were switched (clicking open resulted in closing of the cover and vice versa). Existing automations should be updated. - ([@Ernst79] - [#30566]) ([rfxtrx docs]) 
+- __Ring__ - Ring is now configurable via a config entry. This means it will no longer store the username and password and you're able to configure accounts that use 2FA. It's no longer possible to set scan interval, monitored conditions or ffmpeg arguments. The defaults are used. - ([@balloob] - [#30564]) ([ring docs])  
+- __Binary Sensor__ - Update ON/OFF condition and triggers to match documentation and UI. This change inverts the logic of automations relating to binary sensor locks, but only for automations triggered on the lock state (rather than triggering a lock/unlock action).
```

Comment:
Please clarify in first line that this is only for binary sensors with the `lock` device class that are being used in device automations.